internal/installer/elasticagent_docker.go

```diff
@@ -93,12 +93,13 @@ func (i *elasticAgentDockerPackage) Preinstall(ctx context.Context) error {
 	})
 	defer span.End()
 
-	artifact := "elastic-agent"
+	// handle ubi8 images
+	artifact := "elastic-agent" + common.ProfileEnv["elasticAgentDockerImageSuffix"]
```

Comment:
Ideally, this should be provided in any other way, not reading from a common env. @adam-stokes, we should work on abstracting this so that the preInstall phase knows about what to download.